kyaml/openapi/openapi.go

```diff
@@ -30,6 +33,7 @@ type openapiData struct {
 	schemaByResourceType           map[yaml.TypeMeta]*spec.Schema
 	namespaceabilityByResourceType map[yaml.TypeMeta]bool
 	noUseBuiltInSchema             bool
+	olderVersion                   bool
```

Comment:
The name `olderVersion` is pretty confusing. It looks like a version string. Maybe `isOlderVersion` or something.

Reply:
right, begs the question "older than what?"  

all the fields need explanatory comments

Reply:
updated and commented these fields